src/Endpoints/Delegates/HandlesSystem.php

```diff
@@ -32,14 +42,14 @@ public function stats(): array
         return $this->stats->show();
     }
 
-    public function generateTenantToken(string $apiKeyUid, $searchRules, ?array $options = []): string
+    public function generateTenantToken(string $apiKeyUid, $searchRules, array $options = []): string
```

Comment:
This is a breaking change, no? Since we are now making a requirement to call with the array instead of an optional.

Reply:
In a really rare case it can be a BC break, but it's optional already, as it has the default value `$options = []` which will pass `[]` when the developer doesn't pass anything on its own.

But here it leads to another case described here: https://github.com/meilisearch/meilisearch-php/pull/514/files#r1221466672 which would mean that now the code is less fragile and more strict for things that are required to be set, or how they should be set.